lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -68,8 +70,13 @@ def self.commit(message:, files: nil, push: false)
         elsif !files.nil? && !files.empty?
           Action.sh("git", "add", *files)
         end
-        Action.sh("git", "commit", *args, "-m", message)
-        Action.sh("git", "push", "origin", "HEAD") if push
+        begin
+          Action.sh("git", "commit", *args, "-m", message)
```

Comment:
I see that the args are variable, but is it possible to use the `GitHelper::commit` action here as we have in other places?

Reply:
This IS the `GitHelper::commit` method, this code extract you reviewed is part of that method's implementation details 😄

Reply:
🤦